src/platform-includes/debug-symbols-apple/_default.mdx

```diff
@@ -223,3 +120,61 @@ export SENTRY_URL=https://mysentry.invalid/
 ```
 
 </Alert>
+
+## App Store Connect {#appstore-connect}
+
+<Alert level="" title="Source Context">
+
+The App Store Connect integration doesn't support <PlatformLink to="/data-management/debug-files/source-context/">source context</PlatformLink>. To get it to work, choose one of the methods above.
+
+</Alert>
+
+When you use the App Store Connect integration, Sentry will automatically discover and fetch dSYMs for builds as they become available on App Store Connect.
+
+To configure the integration, go to **[Project] > Settings > Debug Files** and add "App Store Connect" in the "Custom Repositories" section.
+
+![Adding an App Store Connect Integration to a project](adding-asc-repository.png)
+
+Once the integration is set up, it will ensure that new builds are detected and fetched within an hour of being available on App Store Connect.
+
+![A fully set-up App Store Connect Integration](asc-repository.png)
+
+<Alert title="Note" level="info">
+
+Organizations using _Developer_ and _Team_ plans are limited to one App Store Connect source per project. To be able to use multiple App Store Connect sources per project, upgrade to a [_Business_ or _Enterprise_ plan](https://sentry.io/pricing/) or contact [sales@sentry.io](mailto:sales@sentry.io).
+
+</Alert>
+
+Setting up an App Store Connect custom repository requires an App Store Connect API Key. Follow the documentation on [Creating API Keys for App Store Connect API](https://developer.apple.com/documentation/appstoreconnectapi/creating_api_keys_for_app_store_connect_api) and provide the _Issuer_, _Key ID_, and _Private Key_. In order to be able to fetch build details, your API Key needs to be assigned the _Developer_ role.
+
+**Invalid Credentials**
+
+If your App Store Connect API key is revoked on the Apple side, the Sentry custom repository using this key's credentials will no longer be able to discover and fetch dSYM files. You'll see the following warnings in Sentry's user interface:
+
+#### `Issue Details`
+
+![Warning to update credentials in an issue's details](issue-details-warning.png)
+
+#### `Project Settings`
+
+![Warning to update credentials in sidebar](sidebar-warning.png)
+
+#### `Custom Repositories`
+
+![Warning to update credentials in Custom Repositories](custom-repositories-warning.png)
+
+When using Fastlane's _upload_to_testflight_ action, it's recommended to pass the _skip_waiting_for_build_processing_ parameter to speed up the action.
+
+## Bitcode {#dsym-with-bitcode}
+
+The App Store will not accept bitcode submissions from [Xcode 14](https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes). If you’re using Xcode 13, you’ll need to download the dSYMs from App Store Connect after it finishes processing your build, and then upload them to Sentry using one of the methods below:
```

Comment:
```suggestion
The App Store will not accept bitcode submissions from [Xcode 14](https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes). If you’re using Xcode 14, you’ll need to download the dSYMs from App Store Connect after it finishes processing your build, and then upload them to Sentry using one of the methods below:
```

Should it be "if you're using Xcode 14?"

Reply:
No, it should be 13. If you use Xcode 14, it doesn't work.